Examples/LJ_langevin.py

```diff
@@ -43,26 +56,32 @@
 # build the neighbor list from the sampler state
 nbr_list.build_from_state(sampler_state)
 
-from chiron.reporters import _SimulationReporter
+from chiron.reporters import LangevinDynamicsReporter
 
 # initialize a reporter to save the simulation data
 filename = "test_lj.h5"
 import os
 
 if os.path.isfile(filename):
     os.remove(filename)
-reporter = _SimulationReporter("test_lj.h5", lj_fluid.topology, 1)
+reporter = LangevinDynamicsReporter(
+    "test_lj.h5",
+    1,
+    lj_fluid.topology,
+)
 
 from chiron.integrators import LangevinIntegrator
 
 # initialize the Langevin integrator
-integrator = LangevinIntegrator(reporter=reporter, report_frequency=100)
+integrator = LangevinIntegrator(
+    reporter=reporter, report_frequency=100, reinitialize_velocities=True
+)
 print("init_energy: ", lj_potential.compute_energy(sampler_state.x0, nbr_list))
 
-integrator.run(
+updated_sampler_state, updated_nbr_list = integrator.run(
```

Comment:
Following issue #24, we updated the code to avoid updating the samplerstate in place. 
 The MCMCSampler no longer updates in place, but I think there was a weird merge from the upstream that ended up reverting it when you were reviewing.